### DISCUSSION TITLE
What is the purpose of files under Examples?

### DISCUSSION THREAD

**Discussion by hmishfaq** [rlberry-py/rlberry#48] [Q&A] [answered]
What's the purpose/use cases for the files under https://github.com/rlberry-py/rlberry/tree/main/examples?

It would be great to have some examples on how to use them

**Comment by xuedong** [comment, marked as answer]
These are some more thorough examples to showcase different functions of the library and also served as sanity checks when we were implementing the library. And they could be useful for those who want to contribute to get a better understanding of the structure of the library. 

Also, we do plan to turn some of them into more comprehensive and readable jupyter notebook tutorials in the near future.
